.github/workflows/db_check.yml

```diff
@@ -1,4 +1,4 @@
-name: Submitty DB Check
+name: DB Check
```

Comment:
Is there a reason why this check isn't part of the regular CI workflow?  I don't really see a need for this to only run on PRs which touch migrations, which I believe is the only reason why this is in a separate file.

Reply:
I'm not sure why it's like that. We can change it if desired.

Reply:
Moved it into ci.yml.